package-lock.json

```diff
@@ -13,7 +13,10 @@
         "@emotion/react": "^11.10.4",
         "@emotion/styled": "^11.10.4",
         "framer-motion": "^7.4.0",
+        "install": "^0.13.0",
```

Comment:
ignorar este file

Reply:
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5